docker/start.sh

```
@@ -0,0 +1,94 @@
+fi
+
+cleanup() {
+  trap "" EXIT
+
+  if [ -e ./config.sh ]; then
+    print_header "Cleanup. Removing Azure Pipelines agent..."
+
+    # If the agent has some running jobs, the configuration removal process will fail.
+    # So, give it some time to finish the job.
+    while true; do
+      ./config.sh remove --unattended --auth "PAT" --token $(cat "${AZP_TOKEN_FILE}") && break
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)